OpenRA.Mods.Common/Traits/World/EditorActorLayer.cs

```diff
@@ -186,6 +186,10 @@ void SyncMultiplayerCount()
 				Players.Players.Add(pr.Name, pr);
 				worldRenderer.UpdatePalettesForPlayer(pr.Name, pr.Color, true);
 			}
+
+			var creeps = Players.Players.Keys.FirstOrDefault(p => p == "Creeps");
+			if (string.IsNullOrEmpty(creeps))
```

Comment:
Yeah, it probably should, shouldn't it? That's what I get for doing last-minute changes.